package.json

```diff
@@ -187,7 +187,7 @@
     "react-test-renderer": "^15.5.4",
     "rewire": "2.2.0",
     "should": "^11.1.1",
-    "sinon": "^1.17.1",
+    "sinon": "^4.0.2",
```

Comment:
We should make sure there are no wider issues related to this change; 3 versions is quite wide.

Reply:
I agree that it's not necessary right now, but I don't see why not? Their guides tell you which changes need to be made which I've followed:
http://sinonjs.org/releases/v4.0.0/

The main one that's used in Force is seen when moving to [3.0.0](http://sinonjs.org/guides/migrating-to-3.0) which removes `sinon.stub(object, "method", func)`. If the methods don't exist they throw errors which makes things easier to find.